src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+{% import "components/css.html" as css %}
+{% extends "base.html" %}
+{% block title%}New England Organizing{% endblock %}
+{% block content %}
+  <div class="container">
+    <div class="card">
+      <div class="card-body">
+        <div class="card-title">
+          <h1>New England Organizing</h1>
+        </div>
+        <div class="card-text">
+                <p>If you’re looking to organize a cube competition in New England, you’re in the right place! Please read this <strong>entire</strong> page before getting in touch with, as it’s important that you understand exactly what organizing involves.</p>
+                <p><a href="https://docs.google.com/spreadsheets/d/1-qjvGF3loVR0sr-AaDGZE2C7YgwcC1dP2wdPc3BT8fM/edit?usp=sharing">Here</a>’s a link to the competition organization spreadsheet. It will be discussed later on this page. Please go to this page, go to File &gt; Make a Copy, and rename it to your competition.</p>
```

Comment:
Seems like the first sheet of that is mostly aimed at making the schedule for the WCA site, this isn't as necessary anymore since we have the scheduling tool, should this be updated?